catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -98,7 +98,11 @@ function PackageMetaSection({ meta, ...props }: PackageMetaProps) {
             </M.TableRow>
           )}
           {userMeta && (
-            <M.TableRow className={classes.row}>
+            <M.TableRow
+              className={classes.row}
+              data-key="user_meta"
+              data-value={JSON.stringify(userMeta)}
```

Comment:
Probably worth memoize this if `userMeta` is big

Reply:
maybe, but seems like premature optimization at the moment